src/pages/safe/transactions/queue.tsx

```diff
@@ -6,8 +6,28 @@ import { Breadcrumbs } from '@/components/common/Breadcrumbs'
 import TransactionsIcon from '@/public/images/sidebar/transactions.svg'
 import NavTabs from '@/components/common/NavTabs'
 import { transactionNavItems } from '@/components/sidebar/SidebarNavigation/config'
+import { trackEvent } from '@/services/analytics/analytics'
+import { TX_LIST_EVENTS } from '@/services/analytics/events/txList'
+import { useEffect } from 'react'
 
 const Queue: NextPage = () => {
+  const { loading, page } = useTxQueue()
```

Comment:
Please don't call it here. There's a component `PaginatedTxns` that has all cumulatively loaded txns, so you also get a better count.